src/server/accounting.c

```diff
@@ -1164,21 +1124,25 @@ account_jobstr2(job *pjob, int type)
 
 		nd = 0; /* compute total size needed in buf */
 		pal = GET_NEXT(attrlist);
-		while (pal != NULL) {
+		while (pal != NULL)
+		{
```

Comment:
Put the { back up on the line with the while().  It looks like a lot of {'s in this function were moved.  Move them back.

Reply:
I'm suffering from:
https://github.com/Microsoft/vscode/issues/49662 :(

Will reformat it, thanks